tasks.c

```diff
@@ -5726,6 +5772,12 @@ static void prvResetNextTaskUnblockTime( void )
             return xReturn;
         }
     #else /* #if ( configNUMBER_OF_CORES == 1 ) */
+        /* MISRA Ref 8.5.1 [External function shall be declared once.] */
+        /* More details at: https://github.com/FreeRTOS/FreeRTOS-Kernel/blob/main/MISRA.md#rule-85 */
+        /* MISRA Ref 8.6.1 [External function shall be declared once.] */
```

Comment:
Rule 8.6 should be
"An identifier with external linkage shall have exactly one external  definition"
This function is declared once only in task.h. Why are we getting this error.

Reply:
I think it's false alarm here.
xTaskGetCurrentTaskHandle() is only declared at task.h once but there are violations on result.

> (1) Event misra_c_2012_rule_8_6_violation:	Function "xTaskGetCurrentTaskHandle" is declared but never defined.
> 5767 	        TaskHandle_t xTaskGetCurrentTaskHandle( void )

> (1) Event misra_c_2012_rule_8_5_violation:	Symbol "xTaskGetCurrentTaskHandle" is declared more than once.
> Also see events:	[other_declaration]
> 5767 	        TaskHandle_t xTaskGetCurrentTaskHandle( void )

Reply:
Update: The function name is different in task.h/tasks.c because portUSING_MPU_WRAPPERS is set to 1 in coverity test.
Update MISRA.md to descripbe this scenario.